.github/workflows/publish.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: joshmoore/spec-prod@0_3
+      - uses: ome/spec-prod@main
```

Comment:
@joshmoore should we tag https://github.com/ome/spec-prod/commit/1bb5c6816c62fbe2d73ade7f2bea9affaee76c54 e.g. as `v3` to get this passing and merged.

We can then bump back to the HEAD of the spec-prod branch when we copy the content under the `0.4` folder.

Reply:
Done.